test/utils/behaviours.ts

```diff
@@ -198,6 +198,6 @@ export const fnShouldOnlyBeCallableByGovernance = (
   function callFunction(impersonator: Impersonator) {
     const argsArray: unknown[] = typeof args === 'function' ? args() : args;
     const fn = delayedContract().connect(impersonator)[fnName] as (...args: unknown[]) => unknown;
-    return fn(...argsArray, { gasPrice: 0 });
+    return fn(...argsArray);
   }
 };
```

Comment:
ey @gas1cent, did you test if this works? since we're using an empty address to test, i'm afraid that it will raise the 'insufficient funds' error. please confirm and would be ready to go!